docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+
+3. Refer to the [Running Example Scripts](#running-example-scripts) section to start the example Druid Realtime node by issuing the following from within your Druid directory:
+
+    ```bash
+    ./run_example_server.sh
+    ```
+
+#### Kafka Setup and Configuration
+1. Download Kafka
+
+    For this tutorial we will [download Kafka 0.8.2.1]
+    (https://www.apache.org/dyn/closer.cgi?path=/kafka/0.8.2.1/kafka_2.10-0.8.2.1.tgz)
+
+    ```bash
+    tar -xzf kafka_2.10-0.8.2.1.tgz
+    cd kafka_2.10-0.8.2.1
+    ```
+
+2. Start Kafka
+
+    **First, launch ZooKeeper** (refer to the [Set up Zookeeper](#set-up-zookeeper) section for details), then start the Kafka server (in a separate console):
+
+    ```bash
+    ./bin/kafka-server-start.sh config/server.properties
+    ```
+
+3. Create a topic named `wikipedia`
+
+    ```bash
+    ./bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic wikipedia
+    ```
+
+4. Launch a console producer for the topic `wikipedia`
+
+    ```bash
+    ./bin/kafka-console-producer.sh --broker-list localhost:9092 --topic wikipedia
+    ```
+
+5. Copy and paste the following data into the terminal where we launched the Kafka console producer in the previous step:
+
+    ```json
+    {"timestamp": "2013-08-31T01:02:33Z", "page": "Gypsy Danger", "language" : "en", "user" : "nuclear", "unpatrolled" : "true", "newPage" : "true", "robot": "false", "anonymous": "false", "namespace":"article", "continent":"North America", "country":"United States", "region":"Bay Area", "city":"San Francisco", "added": 57, "deleted": 200, "delta": -143}
+    {"timestamp": "2013-08-31T03:32:45Z", "page": "Striker Eureka", "language" : "en", "user" : "speed", "unpatrolled" : "false", "newPage" : "true", "robot": "true", "anonymous": "false", "namespace":"wikipedia", "continent":"Australia", "country":"Australia", "region":"Cantebury", "city":"Syndey", "added": 459, "deleted": 129, "delta": 330}
+    {"timestamp": "2013-08-31T07:11:21Z", "page": "Cherno Alpha", "language" : "ru", "user" : "masterYi", "unpatrolled" : "false", "newPage" : "true", "robot": "true", "anonymous": "false", "namespace":"article", "continent":"Asia", "country":"Russia", "region":"Oblast", "city":"Moscow", "added": 123, "deleted": 12, "delta": 111}
+    {"timestamp": "2013-08-31T11:58:39Z", "page": "Crimson Typhoon", "language" : "zh", "user" : "triplets", "unpatrolled" : "true", "newPage" : "false", "robot": "true", "anonymous": "false", "namespace":"wikipedia", "continent":"Asia", "country":"China", "region":"Shanxi", "city":"Taiyuan", "added": 905, "deleted": 5, "delta": 900}
+    {"timestamp": "2013-08-31T12:41:27Z", "page": "Coyote Tango", "language" : "ja", "user" : "stringer", "unpatrolled" : "true", "newPage" : "false", "robot": "true", "anonymous": "false", "namespace":"wikipedia", "continent":"Asia", "country":"Japan", "region":"Kanto", "city":"Tokyo", "added": 1, "deleted": 10, "delta": -9}
+    ```
+
+#### Finally
```

Comment:
Nm I just noticed that you copy and pasted this from some other page. So it's fine if you don't want to make these improvements now. But I still think they'd be cool